fr.json

```diff
@@ -124,10 +124,10 @@
             "error-trending" : "Unable to load trending items right now<br />Please try again in a few moments.",
             "button-trending" : "View more trending items &rarr;",
             "title-all" : "Tout les Items",
-            "line1-all" : "We have a list of all items in Fortnite: Battle Royale. You can search or filter through them on a dedicated page, [here](/list).",
+            "line1-all" : "Sur fnbr.co tout les items de Fortnite: Battle Royale sont disponibles. Vous pouvez recherchez ou les filtrez sur leurs pages dédiés, [here](/list).",
```

Comment:
Sorry for the long delay! 🙁 Thank you for your contribution!

Is there a particular reason why you seem to translate `we` to `fnbr.co`? Also, it's a single dedicated page and not several dedicated pages.